docs/docs/api/ProxyAgent.md

```diff
@@ -17,6 +17,8 @@ Returns: `ProxyAgent`
 Extends: [`AgentOptions`](Agent.md#parameter-agentoptions)
 
 * **uri** `string | URL` (required) - The URI of the proxy server.  This can be provided as a string, as an instance of the URL class, or as an object with a `uri` property of type string.
+If the `uri` is provided as a string or as a `uri` property of type string, it is then parsed into a URL object according to the [WHATWG URL Standard](https://url.spec.whatwg.org).
+For detailed information on the parsing process and potential validation errors, please refer to the ["Writing" section](https://url.spec.whatwg.org/#writing) of the WHATWG URL Standard.
```

Comment:
```suggestion
If the `uri` is provided as a string or `uri` is an object with an `uri` attribute of type string, then it will be parsed into an URL object according to the [WHATWG URL Standard](https://url.spec.whatwg.org).
For detailed information on the parsing process and potential validation errors, refer to the ["Writing" section](https://url.spec.whatwg.org/#writing) of the WHATWG URL Standard.
```

Reply:
Hey @Uzlopak thank you for the suggestion! Just 2 questions:
- My grammar corrector corrects `an URL` into `a URL`. I even searched it on the web and it seems to be the more correct formula.
- Can we keep the term `property` when referring to the `uri` object?

Based on these considerations and on your suggested changes, I would change the section as follows:

> If the `uri` is provided as a string or `uri` is an object with an `uri` property of type string, then it will be parsed into a URL object according to the [WHATWG URL Standard](https://url.spec.whatwg.org).
> For detailed information on the parsing process and potential validation errors, refer to the ["Writing" section](https://url.spec.whatwg.org/#writing) of the WHATWG URL Standard.

What do you think? Thank you in advance 😄

Reply:
sure

Reply:
Done 😉